src/webviews/mongoClusters/collectionView/CollectionView.tsx

```diff
@@ -325,12 +352,26 @@ export const CollectionView = (): JSX.Element => {
                 </div>
 
                 <QueryEditor
-                    onExecuteRequest={(q: string) =>
+                    onExecuteRequest={(q: string) => {
                         setCurrentContext((prev) => ({
                             ...prev,
                             currrentQueryDefinition: { ...prev.currrentQueryDefinition, queryText: q, pageNumber: 1 },
-                        }))
-                    }
+                        }));
+
+                        trpcClient.common.reportEvent
+                            .mutate({
+                                eventName: 'executeQuery',
+                                properties: {
+                                    ui: 'shortcut',
+                                },
+                                measurements: {
+                                    queryLenth: q.length,
+                                },
+                            })
+                            .catch((error) => {
+                                console.error('Failed to report query event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.